src/main/java/edu/hm/hafner/analysis/parser/GrypeParser.java

```diff
@@ -42,11 +45,17 @@ protected void parseJsonObject(final Report report, final JSONObject jsonReport,
 
     private Issue getIssue(final IssueBuilder issueBuilder, final JSONObject match) {
         JSONObject vuln = match.getJSONObject(VULNERABILIY_TAG);
-        String fileName = match.getJSONObject(ARTIFACT_TAG).getJSONArray(LOCATIONS_TAG).getJSONObject(0)
-                .getString(PATH_TAG);
+        JSONObject artifact = match.getJSONObject(ARTIFACT_TAG);
+        String fileName = artifact.getJSONArray(LOCATIONS_TAG).getJSONObject(0).getString(PATH_TAG);
+        String packageName = artifact.optString(NAME_TAG, "Unknown");
+        String version = artifact.optString(VERSION_TAG);
+        if(version != null){
+            packageName = packageName + " " + version;
+        }
 
         return issueBuilder.setFileName(fileName)
-                .setCategory(vuln.getString(SEVERITY_TAG))
+                .setPackageName(packageName)
```

Comment:
Actually module names are shown as well in the UI (if they are not unique), but I am fine with using package names here.